R/utilities-mass-balance.R

```diff
@@ -38,3 +38,342 @@ envList2PathStringsVector <- function(envList) {
 
   return(pathStringsVector)
 }
+
+
+#' @title plotMeanMassBalance
+#' @description Plot mass balance diagnostics time profiles, cumulative time profiles,
+#' normalized time profiles and cumulative normalized time profiles
+#' @param structureSet `SimulationStructure` R6 class object
+#' @param plotConfigurations List of `PlotConfiguration` R6 class objects
+#' @param massBalanceOptions Options to be defined
+#' (e.g. how to group the plot results, or which molecule to exclude)
+#' @return list of `ggplot` objects
+#' @export
+#' @import tlf
+#' @import ospsuite
+#' @import utils
+plotMeanMassBalance <- function(structureSet,
+                                plotConfigurations = NULL,
+                                selectedCompoundNames = NULL) {
+  simulation <- ospsuite::loadSimulation(structureSet$simulationSet$simulationFile)
+  molecules <- ospsuite::getAllMoleculesMatching(paths = "**", container = simulation)

```

Comment:
WE have a new method here that is going to be at least a factor 100 faster in ospsuite. 
getAllMoleculePaths or somethings`